etl/scripts/faostat/create_new_steps.py

```diff
@@ -81,12 +52,16 @@ def get_channel_from_dag_line(dag_line: str) -> str:
         channel = "garden"
     elif dag_line.startswith("data://meadow/"):
         channel = "meadow"
+    elif dag_line.startswith("data://explorers/"):
+        channel = "explorers"
+    elif dag_line.startswith("snapshot://"):
+        channel = "snapshot"
     elif dag_line.startswith("walden://"):
         channel = "walden"
-    elif dag_line.startswith("grapher://"):
+    elif dag_line.startswith("data://grapher"):
         channel = "grapher"
     else:
-        raise ValueError("dag line not understood")
+        channel = "other"
```

Comment:
Perhaps we could have something like

```python
prefix_to_channel = {
  "data://garden": "garden",
  "data://meadow": "meadow".
...
}

channel = "other"
for prefix, channel_ in prefix_to_channel.items():
   if dag_line.startswith(prefix):
        channel = channel_
        break

Reply:
Thanks Lucas, yes, I think we could recycle some of the stuff in this script, improve it, and use it to create the tool that automatically (or semi-automatically) updates data steps.